core/src/main/java/com/backblaze/b2/client/structures/B2Bucket.java

```diff
@@ -46,13 +49,38 @@ public B2Bucket(String accountId,
                     List<B2CorsRule> corsRules,
                     List<B2LifecycleRule> lifecycleRules,
                     int revision) {
+        this(
+                accountId,
+                bucketId,
+                bucketName,
+                bucketType,
+                bucketInfo,
+                corsRules,
+                lifecycleRules,
+                null,
+                revision
+        );
+    }
+
+    @B2Json.constructor(params = "accountId,bucketId,bucketName,bucketType," +
```

Comment:
as Malay noted above, we can only have on @B2Json.constructor annotation.

Reply:
updated